pkg/clients/security.go

```diff
@@ -0,0 +1,197 @@
+package clients
+
+import (
+	"encoding/base64"
+	"fmt"
+
+	"github.com/bitly/go-simplejson"
+	"github.com/openshift/elasticsearch-proxy/pkg/apis/security"
+	"github.com/openshift/elasticsearch-proxy/pkg/config"
+	log "github.com/sirupsen/logrus"
+)
+
+const (
+	securityIndex = ".security"
```

Comment:
should this be configurable on CLI?

Reply:
I don't believe so.  I actually think we should rename this client and the associated types to be something like 'opendistrosecurity' in the event we can drop the use of it.  We can afford to be opinionated here because i believe the likelyhood of needing to change it to something else is small and adding config options is not trivial.